packages/component-library/stories/README.md

```diff
@@ -0,0 +1,85 @@
+# CIVIC Design System Story Pattern
+
+Storybook is used to document [CIVIC's Design System](https://hackoregon.github.io/civic/). Each component in the library has a story.
+
+As you read through this document, refer to these stories to see the design pattern in use:
+
+- [Horizontal Bar Chart](https://hackoregon.github.io/civic/?path=/story/component-lib-charts-horizontal-bar-chart--standard)
+- [Line Chart](https://hackoregon.github.io/civic/?path=/story/component-lib-charts-line-chart--standard)
+- [Scatterplot](https://hackoregon.github.io/civic/?path=/story/component-lib-charts-scatterplot--standard)
+- [Scatterplot Map](https://hackoregon.github.io/civic/?path=/story/component-lib-maps-scatterplot-map--standard)
+
+Note: To bring up the Addons Panel, select it from the dropdown list in the upper left corner of the screen, next to the CIVIC logo.
+
+## Stories
+
+There are three types of stories:
+
+- Standard
+- Custom
+- Example(s)
+
+### Standard Story
+
+The Standard story is the default configuration of the component. For consistency, this is the configuration that is used unless it doesn't meet the needs of your project.
+
```

Comment:
Would it be useful to say _how_ to determine what the default config is?

Reply:
Very good comment!  Unless there is an issue, with a specification, the standard story will need to be determined when you write the story.  I included some goals and strategies for doing this.